OxfordBodleian/Juel-Jensen/BDLaethg31.xml

```diff
@@ -10,10 +10,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Golgotā, ʾArdǝʾǝt (?), Ṣalota fǝtḥata māy</title>
+                <title xml:lang="gez" xml:id="title1">ጸሎት፡ ዘእግዝእትነ፡ ማርያም፡ በደብረ፡ ጎልጎታ፡,</title>
+                <title xml:lang="gez" corresp="#title1" type="normalized">Ṣalot za-ʾǝgzǝʾǝtǝna Māryām ba-Dabra Golgotā , ʾArdǝʾǝt (?), Ṣalota fǝtḥata māy</title>
+                <title xml:lang="en" corresp="#title1"> Prayer of Our Lady Mary on Mount Golgotha, </title>
```

Comment:
Not doing proofreading, as we discussed last meeting, but is the comma intentional?